### DISCUSSION TITLE
JustPy is not raising exceptions or errors to the console when they occur

### DISCUSSION THREAD

**Discussion by sanket-parte** [elimintz/justpy#668] [General]
JustPy is not raising exceptions or errors to the console when they occur. In order to capture errors, we need to implement exceptional handling. Can someone please tell me if there is any way to capture the errors/exceptions that occur so that I can log them into the console or a file?

**Comment by freemant2000** [comment]
See https://justpy.io/reference/configuration/:

`CRASH = config('CRASH', cast=bool, default=False)`

If set to True, the program terminates if there is an error in an event handler

**Reply by sanket-parte** [reply]
Thank you for your response, I will check this.
